processing/src/main/java/io/druid/segment/NullDimensionSelector.java

```diff
@@ -81,4 +101,136 @@ public int lookupId(String name)
   {
     return Strings.isNullOrEmpty(name) ? 0 : -1;
   }
+
+  @Override
+  public IndexedLongs getLongRow()
+  {
+    return new IndexedLongs()
+    {
+      @Override
+      public int size()
+      {
+        return 1;
+      }
+
+      @Override
+      public long get(int index)
+      {
+        return 0L;
+      }
+
+      @Override
+      public void fill(int index, long[] toFill)
+      {
+
+      }
+
+      @Override
+      public int binarySearch(long key)
+      {
+        return 0;
+      }
+
+      @Override
+      public int binarySearch(long key, int from, int to)
+      {
+        return 0;
+      }
+
+      @Override
+      public void close() throws IOException
+      {
+
+      }
+    };
+  }
+
+  @Override
+  public Comparable getExtractedValueLong(long val)
+  {
+    return val;
+  }
+
+  @Override
+  public IndexedFloats getFloatRow()
+  {
+    throw new UnsupportedOperationException("AAAAA");
```

Comment:
@fjy I had the exception there for testing/debugging, I've restored the commented out block